### PR TITLE
TECHPUBS-3013: Added password provisioning for CEC, and updating the …

### PR DESCRIPTION
**Pull request by jimostrom-hpe**
…default credential in CSM procedures in security_and_authentication

### CODE REVIEW

**Comment by rsjostrand-hpe** [comment]
@jimostrom-hpe I don't believe this PR can merged yet. @nieuwsma uncovered a Gap in our procedure where if we change all of the mountain credentials on the Hardware and CSM side, the Slingshot fabric manager is managing its own set of credentials.

**Comment by jimostrom-hpe** [comment]
> the Slingshot fabric manager is managing its own set of credentials.

OK, let me know how I need to change the procedure. Thanks.

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+
+    Example output:
+    ```
+    x1001c0r5b0: HTTPsGetFailed
+    x1001c1s0b0: HTTPsGetFailed
+    x1001c1s0b1: HTTPsGetFailed
+    x1001c2s0b1: DiscoveryStarted
+    ```
+
+    For each Redfish endpoint that is reported use the following to troubleshoot why it is not `DiscoverOK`:
```
Comment (mpkelly-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:239]: Should this be DiscoverOK or DiscoveryStarted?

Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:239]: You are correct. It should be `DiscoverOK` or `DiscoveryStarted`

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+
+   ```bash
+   ncn-m001# cd site-init
+   ncn-m001# git diff
+   ncn-m001# git add customizations.yaml
+   ncn-m001# git commit -m 'Add customizations.yaml from site-init secret'
+   ```
+
+4. Acquire sealed secret keys:
+    ```bash
```
[the remark this answers is not in the document]
Comment (rsjostrand-hpe) [reply operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md]: Fixed

File: operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
+## Updating the Liquid-Cooled EX Cabinet CEC with Default Credentials after a CEC Password Change
+
+This procedure changes the credential for liquid-cooled EX cabinet chassis controllers and node controller (BMCs) used by CSM services after the CECs have been set to a new global default credential. 
+
+**NOTE:** This procedure does not provision Slingshot switch BMCs (RouterBMCs). Slingshot switch BMC default credentials must be changed using the procedures in the Slingshot product documentation. To update Slingshot switch BMCs, refer to "Change Rosetta Login and Redfish API Credentials" in the *Slingshot Operations Guide* (1.6.0).  
+
+This procedure provisions only the default Redfish root account passwords. It does not modify Redfish accounts that have been added after an initial system installation.
```
Comment (rsjostrand-hpe) [inline operations/security_and_authentication/Updating_the_Liquid-Cooled_EX_Cabinet_Default_Credentials_after_a_CEC_Password_Change.md:7]: @jimostrom-hpe The CSM procedure does not provision the redfish root account passwords (unless you count the stateful reset, but that is more the removal of existing creds and letting the CEC cred take presendence).

The CSM procedure just informs CSM of the that the Node and Chassis BMCs have be given new creds.

**Pull request merged**